src/main/java/com/palehorsestudios/ard/environment/Puzzle.java

```diff
@@ -60,9 +60,9 @@ public String getAnswer() {
     public int getDifficultyInt() {
         int result = 1;
         switch (difficulty) {
-            case "easy" -> result = 1;
-            case "medium" -> result = 2;
-            case "hard" -> result = 3;
+            case "easy": result = 1;
```

Comment:
Don't we need to add break; and default?

Reply:
Or we can use enum... I think

Reply:
I always prefer to use ENUMS, but none the less breaks would be necessary. I think to avoid the whole break thing I will replace with enums and if blocks.